README.md

```diff
@@ -28,16 +30,64 @@ Vite:
 npm install --save-dev vite-plugin-gltf
 ```
 
-## Examples
+### 2. Minimal Configuration
 
-The plugin is unopinionated about which optimizations should be applied to your glTF models. Many optimizations are available off the shelf from [`@gltf-transform/functions`](https://gltf-transform.donmccurdy.com/functions.html):
+The plugin is unopinionated about which optimizations should be applied to your glTF models. Here is a minimal configuration that simply applies Draco compression of your assets:
 
 ```js
 // *.config.js
-import gltf from 'rollup-plugin-gltf'; // (a) Rollup
-import gltf from 'vite-plugin-gltf'; // (b) Vite
+import gltf from "rollup-plugin-gltf"; // (a) Rollup
+import gltf from "vite-plugin-gltf"; // (b) Vite
+
+import { draco } from "@gltf-transform/functions";
+
+return {
+  // ...
+  plugins: [
+    gltf({
+      transforms: [draco()],
+    }),
+  ],
+};
+```
+
+See [advanced configuration](#advanced-configuration) for more complex examples.
+
+_SvelteKit:_ it seems that in SvelteKit SSR should be disabled in order to avoid [mysterious `ReferenceError` issues](https://github.com/nytimes/rd-bundler-3d-plugins/issues/14#issuecomment-1410620469).
+
+### 3. Asset Placement
+
+Assets should be placed somewhere in the `src` directory in order for the plugin to find them.
 
-import { dedup, draco, prune, textureResize, mozjpeg, oxipng } from "@gltf-transform/functions";
+_SvelteKit:_ in a SvelteKit project, place your assets somewhere in `src/lib`. Then you can import the asset's post-build url like so:
+
+```svelte
+<script>
+  import sceneUrl from '$lib/assets/scene.glb';
+</script>
+```
+
+### 4. Execution on Build
+
+The plugin will optimize assets _at build time_. This means you will not observe results when running `npm run dev`. To test the optimization, run `npm run build` and `npm run preview`.
```

Comment:
Nit: the current wording is specific to Vite, npm clients, and particular choices of macro names. Maybe we can clarify it as —

```suggestion
The plugin will optimize assets _at build time_. Vite has [several modes](https://vitejs.dev/guide/env-and-mode.html#modes), and optimization occurs in production (`vite build`) and preview (`vite preview`) but not development (`vite`) modes.
```